cpg_pipes/types.py

```diff
@@ -204,3 +174,51 @@ def parse(str_val: str) -> 'SequencingType':
                 f'Available: {list(str_to_val.keys())}'
             )
         return str_to_val[str_val]
+
+
+class AlignmentInput:
+    """
+    Alignment input can be an indexed CRAM file, or a list of FASTQ file pairs.
```

Comment:
Why couldn't it be unindexed?

Reply:
I could add an extra check in the job and index CRAM automatically if index is not available. Just didn't need so far.